Vagrantfile

```diff
@@ -59,6 +59,12 @@ fi
 
 ## install openvswitch and enable ovsdb-server to listen for incoming requests
 #(apt-get install -y openvswitch-switch > /dev/null) || exit 1
+(wget -nv -O ovs-common.deb https://cisco.box.com/shared/static/v1dvgoboo5zgqrtn6tu27vxeqtdo2bdl.deb &&
```

Comment:
I'm creating a vagrant box with this change right now. Without it Jenkins tests are failing :(